<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Narasimharaos-MacBook-Pro.local" name="com.ibm.commerce.cmc.catalogs.testcases.CatalogsHomePageTest" tests="3" failures="1" timestamp="05 Feb 2019 14:09:33 GMT" time="0.716" errors="0">
  <testcase name="catalogHomePageTitleTest" time="0.017" classname="com.ibm.commerce.cmc.catalogs.testcases.CatalogsHomePageTest">
    <failure type="java.lang.AssertionError" message="expected [Catalogs] but found []">
      <![CDATA[java.lang.AssertionError: expected [Catalogs] but found []
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.ibm.commerce.cmc.catalogs.testcases.CatalogsHomePageTest.catalogHomePageTitleTest(CatalogsHomePageTest.java:37)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- catalogHomePageTitleTest -->
  <testcase name="catalogHomePageHeadingTest" time="0.051" classname="com.ibm.commerce.cmc.catalogs.testcases.CatalogsHomePageTest"/>
  <testcase name="navigatetoNewMasterCategoryPageTest" time="0.648" classname="com.ibm.commerce.cmc.catalogs.testcases.CatalogsHomePageTest"/>
</testsuite> <!-- com.ibm.commerce.cmc.catalogs.testcases.CatalogsHomePageTest -->
